server/controller/index.ts

```diff
@@ -1,4 +1,5 @@
 export { AssessmentController } from './assessment'
+export { CycleDataController } from './cycleData'
+export { SettingsController } from './settings'
 export { UserController } from './user'
 export { UserProviderController } from './userProvider'
```

Comment:
@sorja @rfontanarosa should we stop exporting every controller from index? same convention as we agreed with repositories ?

Reply:
yes